src/docker-compose/dc.sh

```diff
@@ -36,6 +38,16 @@ while [ "$1" != "" ]; do
     "postgres" | "postgresql")
         DATABASE=postgres
         ;;
+    "--no-wait")
+        NO_WAIT=true
+        ;;
+    "down" | "up" | "rm" | "kill" | "stop" | "start" | "run" | "restart" | "pull" | "pause" | "create" | "build")
+        if [ "$DC_CMD" != "" ];then
```

Comment:
Is `DC_CMD` newly added here? Should it be in the usage?

Reply:
I don't see how to use the script in the usage (the DC_CMD) anyways. Maybe I am missing something.